app/views/events/_form.html.erb

```diff
@@ -43,6 +43,20 @@
     </div>
     <%=f.error_span(:active) %>
   </div>
+  <div class="form-group">
+    <%= f.label :veranstalter, :class => 'control-label col-lg-2' %>
```

Comment:
I made this label in German to comply with #11 since it's part of the GUI. Nevertheless, I agree that it's inconsistent with the rest. Any ideas for better localization?